src/test/skript/tests/regressions/7081-time in worlds.sk

```diff
@@ -0,0 +1,29 @@
+test "time in worlds":
+
+	set {_w} to world "world"
+	set {_startTime} to time in {_w}
+
```

Comment:
Unsure if it is but we should likely set gamerule doDaylightCycle to false to ensure time doesn't somehow change causing test to fail

Reply:
this is all done in the same tick so it shouldn't. tests are passing :)